staging/src/k8s.io/client-go/transport/transport_test.go

```diff
@@ -255,6 +263,144 @@ func TestNew(t *testing.T) {
 				},
 			},
 		},
+		"nil holders and nil regular": {
```

Comment:
personal opinion, is not easier to read if omit the False args on the test cases?

Reply:
Just a matter of preference for me.  I find it easier to trace field writes in Goland when fields are explicitly set.